packaging/os/yumrepo.py

```diff
@@ -134,6 +174,14 @@
         are allowed. Substitution variables (e.g. C($releasever)) are honored
         here.
       - The list can also be a regular YAML array.
+  ip_resolve:
```

Comment:
I'm aware. More so that I was wondering if maybe any explanation was appropriate, but maybe not.